gradle.properties

```diff
@@ -2,5 +2,7 @@ projectName='blockingqueue'
 packageOrg=ayash
 group=io.ayash
 version=1.0-SNAPSHOT
-ballerinaLangVersion=2201.10.2
+ballerinaLangVersion=2201.10.0
 ballerinaGradlePluginVersion=3.0.0
+
+stdlibTimeVersion=2.4.0
```

Comment:
Why do we need the `time-native` dependency ? If we do not need it we can remove it.